dracut/30ignition/coreos-remove-initramfs-network.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Shell script always should have
`set -euo pipefail`

Reply:
Done, have been testing in https://github.com/coreos/ignition-dracut/pull/78 since its more time-sensitive in RHCOS, will update this right after